app/helpers/dataclips/application_helper.rb

```diff
@@ -28,6 +28,7 @@ def dataclips_insight_config(insight, options = {})
 
     schema = load_dataclip_insight_schema(insight)
     schema_md5 = Digest::MD5.hexdigest(Marshal.dump(schema.to_json))
+    schema_md5 = nil if options['remember_schema_config'] == false
```

Comment:
```suggestion
    if options['remember_schema_config'] 
       schema_md5 = Digest::MD5.hexdigest(Marshal.dump(schema.to_json))
    end
```

Reply:
if we do not have `remember_schema_config ` then we will not remember schema in your example. I would like to have  `remember_schema_config` default as true

Reply:
then you should write in the beginning of method:

```rb
remember_schema_config = options.fetch('remember_schema_config', true)